tests/publiccloud/pcw_target.pm

```diff
@@ -0,0 +1,39 @@
+use base "consoletest";
+use serial_terminal 'select_serial_terminal';
+use transactional qw(process_reboot);
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    record_info('system', script_output('cat /etc/os-release'));
+    record_info('device', script_output('nmcli -t device'));
+
+    assert_script_run('nmcli con add type ethernet con-name "static-ip" ifname ens4 ipv4.method manual ipv4.addresses 10.0.2.20/15 gw4 10.0.2.2');
+    assert_script_run('nmcli con mod static-ip ipv4.dns "10.136.53.53,10.136.53.54,10.100.2.10"');
+    assert_script_run('nmcli con up static-ip ifname ens4');
+    record_info('ip', script_output('ip a'));
+    script_run('ping -c 1 10.0.2.15');
+    script_run('ping -c 1 download.suse.de');
+
+    assert_script_run('curl -f -v ' . autoinst_url . '/data/slenkins/ssh/authorized_keys >> /root/.ssh/authorized_keys');
+    assert_script_run('curl -f -v ' . autoinst_url . '/data/publiccloud/pcw/sshd_config >/etc/ssh/sshd_config');
+
+    assert_script_run('zypper ar http://download.suse.de/ibs/SUSE:/CA/openSUSE_Tumbleweed/ SUSE_CA');
+    assert_script_run('zypper --gpg-auto-import-keys ref');
+    assert_script_run('transactional-update -n -c pkg install ca-certificates-suse htop iftop iotop atop telnet nmap jq git rsync wget parted tcpdump screen sqlite3 fortune iputils nginx dehydrated dehydrated-nginx podman podman-cni-config cockpit cockpit-podman toolbox python3-selinux');
+    process_reboot(trigger => 1);
```

Comment:
This is already done in Ansible.

Reply:
yes I would expect only python3 install here , all other things should happen via ansible script if we currently missing something we need to add this into ansible script not here

Reply:
do you mean python3-selinux? 
why python3 shouldn't be in the other repo?

Reply:
> Try to remove this as a whole.

no we do need to install python on target ( it is not available by default on MicroOS and requirement by ansible ) 

> do you mean python3-selinux?

no I mean that exception installation of python package target should not be changed by the test at all , no addition of extra repos , no extra packages installation . the whole point of this test is to check that **on clean** VM this ansible playbook can create workable system. With set of this `zypper addrepo ...` / `zypper in ...`  you removing "on clean VM"

Reply:
Anton, there is no Python in the list right now. Isn't Python on MicroOS by default? And if it's not, then Ivan will find out latest.

Reply:
python3 is not present by default I think. However this things are moving rather quickly, it's anyways good to install it explicitly, just to ensure it's present.